pkgs/by-name/co/cook-cli/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
```

Small nit, but to stay consistent with all the other packages, this should be in the sasme line.
```suggestion
{ lib,
```